server.go

```diff
@@ -97,6 +97,14 @@ type Config struct {
 	// to the number of CPUs usable by the current process.
 	Concurrency int
 
+	// BaseContext optionally specifies a function that returns
```

Comment:
nit: please merge the next line into this line, so that this first sentence is on one line :)